bentoml/_internal/utils/ngrok.py

```diff
@@ -44,10 +43,10 @@ def _print_url():
             if data["tunnels"]:
                 tunnel = data["tunnels"][0]
                 logger.info(
-                    " * Ngrok running at: %s",
+                    "Ngrok running at: %s",
```

Comment:
why remove `* ` here? it is meant to be consistent with other Uvicorn logs I believe.

Reply:
I think that is the format before, where we don't use uvicorn logs